ee/libcglue/src/glue.c

```diff
@@ -632,27 +633,30 @@ void * _sbrk(size_t incr) {
 #endif
 
 #ifdef F__gettimeofday
-/*
- * Implement in terms of time, which means we can't
- * return the microseconds.
- */
+int _gettimeofday(struct timeval *tv, struct timezone *tz)
+{
+	if (tv == NULL)
+	{
+		errno = EINVAL;
+		return -1;
+	}
 
-time_t ps2time(time_t *t);
+	{
```

Comment:
this bracket is not needed

Reply:
This is just done to ensure that the scope for the variables enclosed is gone when the ending bracket is reached.